compose/ui/ui-unit/src/jsNativeMain/kotlin/androidx/compose/ui/unit/Density.jsNative.kt

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.ui.unit
+
+import androidx.compose.runtime.Stable
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.geometry.Rect
+import androidx.compose.ui.geometry.isSpecified
+
+/**
+ * Convert a [Offset] to a [DpOffset].
+ */
+@Stable
+@ExperimentalUnitApi
```

Comment:
@dima-avdeev-jb, cross-module usage of experimental API adds a requirement for us to maintain backward compatibility.

Also, they belong to common code, and we can have a conflict in the future when we or Google add them there.

Let's just move them to `ui` and make them internal.

Reply:
@igordmn 
Good idea!
With same package ```package androidx.compose.ui.unit``` ?

Reply:
> With same package package androidx.compose.ui.unit

Yeah, we can have the same package. If we will have a conflict, Compiler will tell us.

Reply:
Made it here:
https://github.com/JetBrains/androidx/pull/364